tests/actions.go

```diff
@@ -447,7 +447,7 @@ func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 	}
 
 	// deploy statefulset webhook and configuration to hijack update statefulset opeartion
-	cmd = fmt.Sprintf("kubectl apply -f %s/webhook.yaml", oa.manifestPath(info.Tag))
+	cmd = fmt.Sprintf(`sed 's/apiVersions: \["v1beta1"\]/apiVersions: ["v1", "v1beta1"]/' %s/webhook.yaml | kubectl apply -f -`, oa.manifestPath(info.Tag))
```

Comment:
`manifests/webhook.yaml` has been modified, so can you omit this operation?

Reply:
no, `v1.0.1` or prior only has one `v1beta1`. We must change them too in the stability test case.